src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -25,27 +26,30 @@
 import io.cucumber.java.en.Then;
 
 public class ConditionsSteps extends Steps {
-	
-	private ConditionsPage conditionsPage;
-	private String patientDashboardId;
-	private By addNewCondition = By.id("conditionui-addNewCondition");
-	private ConditionPage conditionPage;
-
-	@Before(RunTest.HOOK.SELENIUM_DASHBOARD)
-	public void visitDashboard() {
-		initiatePatientDashboard();
-	}
-
-	@After(RunTest.HOOK.SELENIUM_DASHBOARD)
-	public void destroy() {
-		quit();
-	}
-
+    private ConditionsPage conditionsPage;
+    private String patientDashboardId;
+    private By addNewCondition = By.id("conditionui-addNewCondition");
+    private ConditionPage conditionPage;
+    private TestData.PatientInfo testPatient;
+
+    @Before(RunTest.HOOK.SELENIUM_CONDITION)
+    public void visitDashboard() {
+        testPatient = createTestPatient();
+        initiateWithLogin();
+        findPatientPage = homePage.goToFindPatientRecord();
+        findPatientPage.enterPatient(testPatient.identifier);
+        dashboardPage = findPatientPage.clickOnFirstPatient();
+    }
+ 
+    @After(RunTest.HOOK.SELENIUM_CONDITION)
+    public void destroy() {
+        deletePatient(testPatient);
+        quit();
+    }
+    
 	@Given("User clicks on Conditions from Patient dashboard")
 	public void launchManageConditions() {
```

Comment:
Fix the identation to be same . You have two factor identation format, cross check and confirm especially the before and after hook and other methods definition.

Reply:
@sherrif10 i have fixed the indention in the commit https://github.com/openmrs/openmrs-contrib-qaframework/pull/230/commits/34fc2eba733c5a1ecf2e395ef33ec2d6cbd11cb0